chrome_manifest_v3/model.js

```
@@ -377,12 +371,14 @@ Gmail2Trello.Model.prototype.Uploader.prototype = {
       this.parent.newCard.id = new_id_k;
       this.cardId = new_id_k;
     }
-    if (add_title_k && add_title_k.length > 0) {
+    if (add_title_k?.length > 0) {
       this.parent.newCard.title = add_title_k;
     }
   },
 
   attach: function (method, property, upload1, success, failure) {
+    const self = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)